filebeat/docs/overview.asciidoc

```diff
@@ -5,14 +5,28 @@
 <titleabbrev>Overview</titleabbrev>
 ++++
 
-Filebeat is a log data shipper for local files. Installed as an agent on your servers, Filebeat monitors the log directories or specific log files, tails the files,
-and forwards them either to https://www.elastic.co/products/elasticsearch[Elasticsearch] or https://www.elastic.co/products/logstash[Logstash] for indexing.
+{beatname_uc} is a lightweight shipper for forwarding and centralizing log data.
+Installed as an agent on your servers, {beatname_uc} monitors the log
+files or locations that you specify, tails the logs, and forwards log data
+to either to https://www.elastic.co/products/elasticsearch[Elasticsearch] or
+https://www.elastic.co/products/logstash[Logstash] for indexing.
 
-Here's how Filebeat works: When you start Filebeat, it starts one or more prospectors that look in the local paths you've specified for log files. For each log file that the prospector locates, Filebeat starts a harvester. Each harvester reads a single log file for new content and sends the new log data to libbeat, which aggregates the events and sends the aggregated data to the output that you've configured for Filebeat.
+// REVIEWERS: Is it still accurate to say "tails the logs" or is that a file
+// concept that doesn't make sense for some of the newer inputs that we've added.
```

Comment:
kind of depends on the input type. In a 'pull' based input, we will try to 'tail' in a sense of trying to collect new logs only. We also have 'push' based inputs (e.g. syslog or plain UDP/TCP), this would make sense for.

How about 'collect'. Instead of `logs` we might have `events` or `documents`. With introduction of UDP/TCP, we already widened the scope.

Reply:
I've made this description more generic.